src/Orc.NuGetExplorer.Xaml/Windows/Service/AnimationService.cs

```diff
@@ -17,7 +17,7 @@ public AnimationService()
 
         public Storyboard GetFadeInAnimation(DependencyObject dependencyObject)
         {
-            var sb = (_resourceDictionary["FastFadeIn"] as Storyboard)?.Clone();
+            var sb = ((Storyboard)_resourceDictionary["FastFadeIn"]).Clone();
```

Comment:
This seems to be an unsafe change?

Reply:
I am expecting to throw exception here if something broken